scripts/david-adediji.py

```diff
@@ -0,0 +1,5 @@
+Fullname='David Adediji'
+ID='HNG-05328'
+Email='davidadediji@gmail.com'
+Language='Python'
+print(f'Hello World, this is {Fullname} with HNGi7 ID {ID} and email {Email} using {Language} for stage 2 task')
```

Comment:
advised to implement without the `f` in the print()

Reply:
done. Thanks

Reply:
suggested correction: 
`print('Hello World, this is {} with HNGi7 ID {} and email {} using {} for stage 2 task').format(fullname, ID, Email, Language)`